main.go

```diff
@@ -94,7 +95,6 @@ func main() {
 		Desc:   "Throttle",
 		EnvVar: "THROTTLE"})
 
-	//TODO can we use custom headers
 	messageTypeEndpointsMap := map[string]string{
```

Comment:
Yeah the app is a lot less complex than when I originally wrote this stuff. Will take a look at refactoring tomorrow